app/scripts/modules/amazon/src/serverGroup/details/rollback/rollbackServerGroup.html

```diff
@@ -76,6 +89,29 @@
           </p>
         </div>
       </div>
+      <div class="row" ng-if="command.rollbackType === 'PREVIOUS_IMAGE'">
+        <div class="col-sm-11 col-sm-offset-1">
+          <ol>
+            <li>Deploy <em>{{ previousServerGroup.imageId }}</em> [
+              <strong>min</strong>: {{serverGroup.capacity.desired}},
+              <strong>max</strong>: {{ serverGroup.capacity.max }},
+              <strong>desired</strong>: {{ serverGroup.capacity.desired }}
+              ]<br/>(minimum capacity pinned at {{serverGroup.capacity.desired}} to prevent autoscaling down during rollback)
+            <li ng-if="command.rollbackContext.targetHealthyRollbackPercentage < 100">
+              Wait for at least {{minHealthy(command.rollbackContext.targetHealthyRollbackPercentage)}}
+              instances to report as healthy
+            </li>
+            </li>
+            <li>Disable {{ serverGroup.name }}</li>
+            <li>Restore minimum capacity of <em>new server group</em> [
+              <strong>min</strong>: {{ serverGroup.capacity.min }}
+              ]</li>
+          </ol>
+          <p>
+            This rollback will affect server groups in {{ serverGroup.account }} ({{ serverGroup.region }}).
+          </p>
```

Comment:
Why is this sentence only being displayed if we're rolling back to the previous image and not explicitly selecting an image?

Reply:
The steps are a little different depending on the strategy so not _too_ much duplication. 

It would seem more confusing to me to try and `ng-if` each descriptive step?

Reply:
Sorry, not suggesting interleaving it with `ng-if`s. Duplication makes sense here - keeps it readable.

Reply:
Good ... I thought you were talking some crazy right there.